tests/tracing.spec.ts

```diff
@@ -103,6 +103,21 @@ test('should collect two traces', async ({ context, page, server }, testInfo) =>
   }
 });
 
+test('should not stall on dialogs', async ({ page, context, server }) => {
+  await context.tracing.start({ screenshots: true, snapshots: true });
+  await page.goto(server.EMPTY_PAGE);
+
+  page.on('dialog', async dialog => {
+    await dialog.accept();
```

Comment:
Do we have to catch this promise and await it? I am worried that `dialog.accept` will race with `context.close` and throw an unhandled promise rejection.

Reply:
I believe we are ok since the following call to confirm is blocking and is serialized with the accept ack.